_pages/rfc-pairing.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: pageminimal
+title: "RFC [r] - WeAllJS #pairing"
+permalink: /rfc-pairing
+---
+
+# RFC [r] - WeAllJS #pairing
+
+[ratified 2016-09-17]
+{: text-italic}
```

Comment:
Will this be formatted correctly? Clicking 'View' shows this line literally and doesn't italicize the previous line.